mautic/mautic
# The tab Permissions should be hidden when "Has full system access?" is set to Yes

Fixes #13441 by @andersonjeccel 


| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


Hide the "Permissions" tab in the role creation interface when "Has full system access?" is set to "Yes". This prevents confusion and ensures a smoother user experience by removing access to an irrelevant tab.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open mautic and open the right sidebar
3. Click on Roles
4. Create a new role (+ New)
4. Set "Has full system access?" to "Yes" and click save.
5. Verify that the "Permissions" tab is hidden.
6. Set "Has full system access?" to "No" and click save and verify that the "Permissions" tab becomes visible again.

#### Screenshots:
<img width="1208" alt="image" src="https://github.com/mautic/mautic/assets/95358802/82e018b0-4cd6-42d2-8d69-64b390a72eaf">
<img width="1208" alt="image" src="https://github.com/mautic/mautic/assets/95358802/f5b724cd-28f7-4dce-9bb6-37c1db61ca89">
